src/browser/BrowserOptionDialog.cpp

```diff
@@ -120,6 +122,7 @@ void BrowserOptionDialog::loadSettings()
     m_ui->updateBinaryPath->setChecked(settings->updateBinaryPath());
     m_ui->chromeSupport->setChecked(settings->chromeSupport());
     m_ui->chromiumSupport->setChecked(settings->chromiumSupport());
+    m_ui->braveSupport->setChecked(settings->braveSupport());
```

Comment:
I wasn't sure whether this should do inside or outside the `ifndef Q_OS_WIN` block beginning on line 127. I'm not clear on the purpose of that block.

Reply:
It goes inside the `#ifndef Q_OS_WIN` block because non-Chromium settings are used when it's not Windows.

Reply:
Done.